IFTPipeline.jl/test/test-feature-extract.jl

```diff
@@ -25,15 +25,27 @@ end
 @testset "feature-extraction.jl" begin
     image_size = (8, 8)
     
-    # Can cast and uncast all sizes of integer
+    # Can cast and uncast all sizes of unsigned integer
     test_cast_uncast(rand(UInt8, image_size))
     test_cast_uncast(rand(UInt16, image_size))
     test_cast_uncast(rand(UInt32, image_size))
     test_cast_uncast(rand(UInt64, image_size))
 
-    # Can load all sizes of fixed point integer:
+    # ... and signed integer
+    test_cast_uncast(rand(Int8, image_size))
+    test_cast_uncast(rand(Int16, image_size))
+    test_cast_uncast(rand(Int32, image_size))
+    test_cast_uncast(rand(Int64, image_size))
+
+    # Can load all sizes of unsigned integer:
     test_save_load(rand(UInt8, image_size))
     test_save_load(rand(UInt16, image_size))
     test_save_load(rand(UInt32, image_size))
     test_save_load(rand(UInt64, image_size))
+
+    # ... and signed integer
+    test_save_load(rand(Int8, image_size))
+    test_save_load(rand(Int16, image_size))
+    test_save_load(rand(Int32, image_size))
+    test_save_load(rand(Int64, image_size))
```

Comment:
Because image size is not changing, one can simplify this something like
```julia
_test_save_load(t) = test_save_load(rand(t, image_size))
```
Then just call _test_save_load passing the type only. For example, 
```julia
_test_save_load(Int8)
```